packages/web-react/src/components/Grid/README.md

```diff
@@ -26,4 +26,44 @@ Use Grid to build multiple column layouts. This Grid works on twelve column syst
 | `UNSAFE_className` | `string`                                  | —       | ✕        | Wrapper custom class name           |
 | `UNSAFE_style`     | `CSSProperties`                           | —       | ✕        | Wrapper custom style                |
 
-For detailed information see [Grid](https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web/src/scss/components/Grid/README.md) component
+For detailed information see [Grid] component
+
+## GridSpan
+
+Grid Span allows to center content over multiple grid columns.
+
+For more info when and why to use it, see [Grid] component.
+
+Basic example usage:
+
+```jsx
+<Grid>
+  <GridSpan over="2">2</GridSpan>
+  <GridSpan over="4">4</GridSpan>
+  <GridSpan over="6">6</GridSpan>
+  <GridSpan over="8">8</GridSpan>
+  <GridSpan over="10">10</GridSpan>
+</Grid>
+```
+
+Advanced example usage:
+
+```jsx
+<Grid elementType="ul">
+  <GridSpan elementType="li" over="10" tablet="8" desktop="6">
+    {' '}
```

Comment:
why the spaces here?

Reply:
spaces removed

Reply:
Really? I still see `{'  '}`. Was the comment about this?

Reply:
now it should be finally gone